db/migrate/20170503212744_drop_user_table.rb

```diff
@@ -0,0 +1,5 @@
+class DropUserTable < ActiveRecord::Migration
```

Comment:
Since this PR was never merged, you can just remove all these migrations (and that way you don't have to create a table just to remove it in a later migration).